x-pack/platform/plugins/shared/stack_connectors/public/connector_types/webhook/webhook.tsx

```diff
@@ -30,17 +31,26 @@ export function getConnectorType(): ConnectorTypeModel<
       defaultMessage: 'Webhook data',
     }),
     validateParams: async (
-      actionParams: WebhookActionParams
+      actionParams: WebhookActionParams,
+      connectorConfig?: WebhookConfig
     ): Promise<GenericValidationResult<WebhookActionParams>> => {
+      const webhookMethod: WebhookMethods = connectorConfig?.method
+        ? connectorConfig.method
+        : WebhookMethods.POST;
```

Comment:
```suggestion
      const webhookMethod: WebhookMethods | null = connectorConfig?.method
        ? connectorConfig.method
        : null;
```

Otherwise, we make an assumption that the config of the connector is set as POST, which may change in the future.